### PR TITLE
Roll out next 36.20220325.1.0

### PR DESCRIPTION
**Pull request by ravanelli**
next
    version: 36.20220325.1.0 (latest)
    start: 2022-03-29 14:00:00+00:00 UTC (in 13:32:54)
           2022-03-29 10:00:00-04:00 Raleigh/New York/Toronto
           2022-03-29 16:00:00+02:00 Berlin/France/Poland
    duration: 2880m (48.0h)

Signed-off-by: Renata Andrade Matos Ravanelli <rravanel@redhat.com>

### CODE REVIEW

**Comment by dustymabe** [comment]
the check stream diffs GH action is showing an unexpected change but I can't tell what it is complaining about (definitely some room for improvement there on the GH action).

I suspect maybe it has to do with the `v0.2.3-1-g50a1837` for the stream generator version.. @ravanelli - can you check your local build and see if that can get fixed?

**Comment by dustymabe** [comment]
> the check stream diffs GH action is showing an unexpected change but I can't tell what it is complaining about (definitely some room for improvement there on the GH action).

Oh. I see. It shows up in the diff view. 
> 
> I suspect maybe it has to do with the `v0.2.3-1-g50a1837` for the stream generator version.. @ravanelli - can you check your local build and see if that can get fixed?

yep that seems to be it.

File: updates/next.json [hunk 1]
```diff
@@ -45,19 +45,19 @@
...
     },
     {
-      "version": "35.20220313.1.0",
+      "version": "36.20220325.1.0",
       "metadata": {
         "rollout": {
           "duration_minutes": 2880,
-          "start_epoch": 1647439200,
+          "start_epoch": 1648562400,
           "start_percentage": 0.0
```
Comment (dustymabe) [inline updates/next.json:61]: we need to add a barrier here - look at https://github.com/coreos/fedora-coreos-streams/commit/f1969b2 for an example

**Comment by bgilbert** [comment]
> Oh. I see. It shows up in the diff view.

You can also get a complete list on the workflow summary page for the run ([example](https://github.com/coreos/fedora-coreos-streams/actions/runs/2055783100)).

**Review by HuijingHei — APPROVED** [review]

**Review by bgilbert — APPROVED** [review]

**Pull request merged**